src/test/evo_trivialvalidation.cpp

```diff
@@ -27,13 +27,12 @@ BOOST_AUTO_TEST_CASE(trivialvalidation_valid)
 
     for (uint8_t idx = 1; idx < vectors.size(); idx++) {
         UniValue test = vectors[idx];
-        uint32_t txHeight;
         uint256 txHash;
         std::string txType;
         CMutableTransaction tx;
         try {
             // Additional data
-            txHeight = test[0].get_int();
+            [[maybe_unused]] uint32_t txHeight = test[0].get_int();
```

Comment:
why do we have it then?

Reply:
because test[0] can be not int at all. I re-wrote it a bit in new version